app/src/main/java/de/westnordost/streetcomplete/quests/bus_stop_name/AddBusStopNameForm.kt

```diff
@@ -20,6 +25,27 @@ class AddBusStopNameForm : AAddLocalizedNameForm<BusStopNameAnswer>() {
         AnswerItem(R.string.quest_streetName_answer_cantType) { showKeyboardInfo() }
     )
 
+    // this filter needs to be kept somewhat in sync with the filter in AddBusStopName
+    private val busStopsWithNamesFilter = """
+        nodes, ways, relations with
+        (
+          public_transport = platform and bus = yes
+          or (highway = bus_stop and public_transport != stop_position)
+          or railway = halt
+          or railway = station
+          or railway = tram_stop
```

Comment:
```suggestion
          public_transport = platform and bus = yes
          or highway = bus_stop and public_transport != stop_position
          or railway ~= halt|station|tram_stop
```

(same in `AddBusStopName`)

Reply:
Fair enough. It's just `~`, though.